ocaml/tests/test_daemon_manager.ml

```diff
@@ -145,3 +145,6 @@ let test =
   ; ("test_timeout_succeed", `Slow, test_timeout_succeed)
   ; ("test_timeout_fail", `Slow, test_timeout_fail)
   ]
+
+let () =
+  Alcotest.run "Daemon Manager suite" ["test", test]
```

Comment:
To avoid this 'No newline at end of file', I'd do change this setting in vscode: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
I recommend installing ocamlformat using opam and running `make format` ;)